File: sportsbook-app/risk-service/src/main/java/com/sportsbook/riskservice/controller/SampleController.java
package com.sportsbook.riskservice.controller;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class SampleController {

    @GetMapping("/api/riskservice/ping")
    public String ping() {
        return "risk-service pong";
    }
}

File: sportsbook-app/risk-service/src/main/kotlin/com/sportsbook/riskservice/controller/SampleController.kt
package com.sportsbook.riskservice.controller

import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController

@RestController
class SampleController {

    @GetMapping("/api/riskservice/ping")
    fun ping(): String = "risk-service pong"
}
